### PR TITLE
Use e2eskipper package in test/e2e/auth

### PR DESCRIPTION
**Pull request by YuikoTakada**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

This PR use e2eskipper package instead of using func Skip* in kubernetes/test/e2e/framework/skip.go

**Which issue(s) this PR fixes**:

Ref: #87047 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by k-toyoda-pi** [comment]
It seems that audit.go has a framework.Skipf which can be replaced with skipper around line 658.

File: test/e2e/auth/audit.go [hunk 1]
```diff
@@ -62,7 +63,7 @@ var _ = SIGDescribe("Advanced Audit [DisabledForLargeClusters][Flaky]", func() {
 	f := framework.NewDefaultFramework("audit")
 	var namespace string
 	ginkgo.BeforeEach(func() {
-		framework.SkipUnlessProviderIs("gce")
+		e2eskipper.SkipUnlessProviderIs("gce")
```
Comment (oomichi) [inline test/e2e/auth/audit.go]: line 659 also needs to be replaced.

Comment (oomichi) [reply test/e2e/auth/audit.go]: Ah @k-toyoda-pi already pointed this.

**Comment by YuikoTakada** [comment]
@k-toyoda-pi @oomichi  thank you for your comments. done.

**Review by k-toyoda-pi — COMMENTED** [review]
Thank you for updating.
/lgtm

**Review by SataQiu — COMMENTED** [review]
Thanks @YuikoTakada 
/approve

**Pull request merged**